server/src/test/java/io/spine/server/entity/storage/given/EntityRecordWithColumnsTestEnv.java

```diff
@@ -47,4 +49,12 @@ public EntityWithoutColumns(String id) {
             super(id);
         }
     }
+
+    public static class TestEntityBuilder extends EntityBuilder<TestEntity, String, Project> {
```

Comment:
There are two similar classes in this changeset. Another one is in `TestEntity`. Not sure we need both.